screens/TermsAndConditionsScreen/TermsAndConditionsScreen.js

```diff
@@ -73,10 +82,13 @@ const styles = StyleSheet.create({
     width: 400
   },
   agreement: {
+    alignSelf: 'flex-start',
     fontSize: 20,
     marginLeft: 20
   },
   button: {
+    alignSelf: 'center',
```

Comment:
Ooooh! I didn't know this was a property, sweet!

Reply:
Doesn't do it sometimes, depends on container display property